Sources/Commands/SwiftTestTool.swift

```diff
@@ -488,14 +488,21 @@ final class ParallelTestRunner {
             thread.start()
             return thread
         })
-
-        // Holds the output of test cases which failed.
+        
+        // Whether to output on success
+        let shouldOutputSuccess = Process.verbose
+        
+        // Holds the output of test cases.
         var failureOutput = [String]()
+        var successOutput = [String]()
```

Comment:
We can use just one variable to keep track of the outputs.

Reply:
This will change the ordering of the output, is that okay?

Reply:
Yeah, I don't know if it matters much if someone wants complete output OTOH it is probably nice to keep all the failing tests separated from the succeeded tests. Okay, lets keep them separated!